.gitignore

```
@@ -5,6 +5,9 @@ __pycache__/
 # C extensions
 *.so
 
+# ctags
+.tags
```

I'm using ctags from `universal-ctags` and the default filename is `tags` (without the leading `.`)
Add it as well maybe?

```suggestion
.tags
tags
```